CRM/Core/Form/RecurringEntity.php

```diff
@@ -399,6 +399,7 @@ public static function postProcess($params = array(), $type, $linkedEntities = a
                 'is_active' => 1,
               );
               $excludeDateList[] = $optionGroupValue['value'];
+              // Something is wrong here. The returned BAO object is not being used.
               CRM_Core_BAO_OptionValue::create($optionGroupValue);
```

Comment:
@eileenmcnaughton This does not seem to be correct. However I'm unsure of how to interpret this. Would love your opinion on whether this could be causing the error?